.env.staging.example

```
@@ -0,0 +1,231 @@
+VITE_API_RATE_WINDOW=3600
+
+# Image Optimization
+VITE_ENABLE_IMAGE_OPTIMIZATION=true
+VITE_IMAGE_QUALITY=90
+
+# =============================================================================
+# TESTING CONFIGURATION
+# =============================================================================
+
+# Test Data
+VITE_ENABLE_TEST_DATA=true
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The VITE_ENABLE_TEST_DATA key is duplicated

(DuplicatedKey)